docs/automount_service_account_token_exception.md

```diff
@@ -0,0 +1,50 @@
+# **Knowledge Base Article: Why the OADP Operator Pod Requires `automountServiceAccountToken` to Be Enabled**
+
+---
+
+## **Rationale for Allowing the Default `automountServiceAccountToken` Behavior in OADP Operator Pods**
+
+The Kubernetes Best Practices team recommends setting `automountServiceAccountToken: false` for pods to reduce security risks by minimizing exposure of service account tokens. However, the **OADP Operator Pod** relies on the Kubernetes API server for its critical functions, and disabling the token mount would render it inoperable. While the OADP Operator Pod does not explicitly set `automountServiceAccountToken: true`, it depends on the **default Kubernetes behavior**, which mounts the service account token into the pod.
```

Comment:
It is not clear for me which Pod is `OADP Operator Pod`?

Reply:
let's use the actual pod name as well.

Reply:
Done, updated the PR